babe

```diff
@@ -46,6 +46,8 @@ babeConfig = None
 disableEmail = False
 requestsTimeout = 5
 adminPort = 1
+instancePort = 0
+isNodeAppCache = dict()
```

Comment:
Why is this a global variable? Only constants are global. Everything else should be passed as params

Reply:
Also this is true for all apps, why keep app info cache only for node?